src/casdoor/main.py

```diff
@@ -70,14 +70,51 @@ def get_oauth_token(self, code: str) -> str:
         :param code: the code that sent from Casdoor using redirect url back to your server.
         :return: access_token
         """
+        r = self.oauth_token_request(code)
+        access_token = r.json().get("access_token")
+
+        return access_token
+
+    def oauth_token_request(self, code: str) -> requests.Response:
+        """
+        Request the Casdoor server to get access_token.
+        :param code: the code that sent from Casdoor using redirect url back to your server.
+        :return: Response from Casdoor
+        """
         url = self.endpoint + "/api/login/oauth/access_token"
         params = {
             "grant_type": self.grant_type,
             "client_id": self.client_id,
             "client_secret": self.client_secret,
             "code": code,
         }
-        r = requests.post(url, params)
+        return requests.post(url, params)
+
+    def refresh_token_request(self, refresh_token: str, scope: str = "") -> requests.Response:
+        """
+        Request the Casdoor server to get access_token.
+        :param refresh_token: refresh_token for send to Casdoor
+        :param scope: OAuth scope
+        :return: Response from Casdoor
+        """
+        url = self.endpoint + "/api/login/oauth/refresh_token"
+        params = {
+            "grant_type": self.grant_type,
+            "client_id": self.client_id,
+            "client_secret": self.client_secret,
+            "scope": scope,
+            "refresh_token": refresh_token,
+        }
+        return requests.post(url, params)
+
+    def get_oauth_refreshed_token(self, refresh_token: str, scope: str = "") -> str:
```

Comment:
Rename to `refresh_oauth_token` based on Go's naming: https://github.com/casdoor/casdoor-go-sdk/blob/35bd7afc4c75b7465557c53bb783201b30dcfb02/casdoorsdk/token.go#L53